internal/common/baseclient/iito.go

```diff
@@ -2,43 +2,53 @@ package baseclient
 
 import "github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 
-// RequestGift allows clients to signalize that they want a gift
+// FIXME: Wait for implementation of getClientsAlive()
+const aliveClients = 6
```

Comment:
we don't need this, we can just say 
`requests := shared.GiftRequestDict{}` in L12 & L27 -- maps can be dynamic